app-backend/api/src/main/scala/user/PseudoUsernameService.scala

```diff
@@ -60,20 +60,20 @@ object PseudoUsernameService {
     "Taygete",
     "Thelxinoe",
     "Thyone",
-    "S/2003 J2",
-    "S/2003 J3",
-    "S/2003 J4",
-    "S/2003 J5",
-    "S/2003 J9",
-    "S/2003 J10",
-    "S/2003 J12",
-    "S/2003 J15",
-    "S/2003 J16",
-    "S/2003 J18",
-    "S/2003 J19",
-    "S/2003 J23",
-    "S/2011 J1",
-    "S/2011 J2"
+    "S/2003-J2",
```

Comment:
This has not been reported, but I am worried if `/` could also be a source of potential issue (in email's user name section, and in password). I would suggested to replace `/` with `-` as well. What do you think?

Reply:
I don't have a good intuition about whether or not that'd be an issue so I think this is a good "better safe than sorry" thing. I'll make the change.